torch/mlc/mlc_rmsprop.py

```diff
@@ -0,0 +1,68 @@
+import torch
+from .mlc_optimizer import MLCOptimizer
+from ..optim.rmsprop import RMSprop
+
+
+class MLCRMSprop(MLCOptimizer):
```

Comment:
@smessmer , should the MLC optimizers live on torch.mlc or torch.backends.mlc. Currently, the code is on torch.mlc. but we can move it to another location. Would love to hear your thoughts?

Reply:
@smessmer any thoughts on this?

Reply:
@belmhal what's your email? I'd love to collaborate on this.